src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipeflatlands.java

```diff
@@ -24,7 +25,21 @@ public boolean run(final CommandSender sender, Player playerSender, Command cmd,
             player.kickPlayer("Server is going offline for flatlands wipe, come back in a few minutes.");
         }
 
-        server.shutdown();
+        if (!plugin.cpb.isEnabled())
```

Comment:
Let me have another think about that, Im not convinced my original logic makes sense thinking about it. I'll let you know.